doc/api/synopsis.markdown

```diff
@@ -2,28 +2,42 @@
 
 <!--type=misc-->
 
+`node [options] [v8 options] [script.js | -e "script"] [arguments]`
+
+Please see the [Command Line Options][] document for information about
+different options and ways to run scripts with Node.
+
+## Hello World Example
+
 An example of a [web server][] written with Node.js which responds with
 `'Hello World'`:
 
 ```js
 const http = require('http');
 
-http.createServer( (request, response) => {
-  response.writeHead(200, {'Content-Type': 'text/plain'});
-  response.end('Hello World\n');
-}).listen(8124);
+const hostname = '127.0.0.1';
+const port = 3000;
+
+const server = http.createServer((req, res) => {
+  res.statusCode = 200;
+  res.setHeader('Content-Type', 'text/plain');
+  res.end('Hello World\n');
+});
 
-console.log('Server running at http://127.0.0.1:8124/');
+server.listen(port, hostname, () => {
```

Comment:
`listen()` doesn't get an error argument. have to listen for `server.on('error'`.